### PR TITLE
dev, 3.0: Add backup and restore documentation

### PR DESCRIPTION
**Pull request by aylei**
### What is changed, added or deleted? 
Add backup and restore documentation.

EN: https://github.com/pingcap/docs/pull/1368

### Which version does your change affect? 
`3.0`,`dev`

### Checklist 

- [x] Add a new file to `TOC.md`
- [x] No Tab spaces anywhere 
- [x] Leave a blank line both before and after a code block
- [x] Keep the first level heading consistent with `title` in metadata
- [x] Use *four* spaces for each level of indentation except that in `TOC.md`

### CODE REVIEW

**Comment by aylei** [comment]
@tennix @DanielZhangQD @lilin90 @TomShawn PTAL

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+全量备份使用 `mydumper` 来获取 TiDB 集群的逻辑备份数据。全量备份任务会创建一个 PVC([PersistentVolumeClaim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistentvolumeclaims)) 来存储数据。
+
+默认配置下，备份任务使用 PV ([Persistent Volume](https://kubernetes.io/docs/concepts/storage/persistent-volumes/#persistent-volumes)) 来存储备份数据。你也可以通过修改配置将备份数据存储到 [Google Cloud Storage](https://cloud.google.com/storage/)，[Ceph Object Storage](https://ceph.com/ceph-storage/object-storage/) 或 [Amazon S3](https://aws.amazon.com/s3/) 中，在这种情况下，数据在上传到对象存储前，会临时存储在 PV 中。参考 [Kubernetes 上的 TiDB 集群备份配置](/reference/configuration/tidb-in-kubernetes/backup-configuration.md) 了解所有的配置选项。
+
+你可以配置一个定时执行的全量备份任务，也可以临时执行一个全量备份任务。
+
+### 定时全量备份
+
+定时全量备份是一个与 TiDB 集群一同创建的定时任务，它会像 `crontab` 任务那样周期性地运行。
+Scheduled full backup is created alongside the TiDB cluster, and it runs periodically like the crontab job.
```
Comment (DanielZhangQD) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]: This should be removed.

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+
+你可以修改 TiDB 集群的 `values.yaml` 文件来配置定时全量备份：
+To configure a scheduled full backup, modify the `scheduledBackup` section in the `values.yaml` file of the TiDB cluster:
```
Comment (DanielZhangQD) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]: ditto

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+```shell
+kubectl get pvc -n <namespace> -l app.kubernetes.io/component=backup,pingcap.com/backup-cluster-name=<cluster-name>
+```
+
+假如你将数据存储在 [Google Cloud Storage](https://cloud.google.com/storage/)，[Ceph Object Storage](https://ceph.com/ceph-storage/object-storage/) 或 [Amazon S3](https://aws.amazon.com/s3/) 中，你可以用这些存储系统自带的图形界面或命令行工具查看全量备份。
+
+## 数据恢复
+
+ 使用 `pingcap/tidb-backup` 这个 Helm chart 进行数据恢复，步骤如下：  To perform a restore operation, modify the `values.yaml` file:
```
Comment (DanielZhangQD) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]: English words

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+
+* 修改 `values.yaml`：
+    * 将 `clusterName` 设置为目标 TiDB 集群名；
+    * 将 `mode` 设置为 `restore`；
+    * 将 `name`  设置为用于恢复的备份名字（你可以参考[查看备份](#查看备份)来寻找可用的备份数据）。假如备份数据存储在 [Google Cloud Storage](https://cloud.google.com/storage/)，[Ceph Object Storage](https://ceph.com/ceph-storage/object-storage/) 或 [Amazon S3](https://aws.amazon.com/s3/) 中，你必须保证这些存储的相关配置与执行[全量备份](#全量备份)时一致。
+* 创建一个包含数据库用户名和密码的 Kubernetes [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) 该用户必须拥有数据备份所需的数据库相关权限（假如你在[全量备份](#全量备份)时已经创建了该 Secret，则可以跳过这步）：
```
Comment (DanielZhangQD) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]: how to use this secret?

**Comment by aylei** [comment]
@DanielZhangQD Thanks, seems that I've commit a outdated version😅

**Comment by aylei** [comment]
@DanielZhangQD Addressed, PTAL

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+
+# Kubernetes 上的 TiDB 集群备份与恢复
+
+## 概览
+
+这篇文档详细描述了如何对 Kubernetes 上的 TiDB 集群进行数据备份和数据恢复。
+
+Kubernetes 上的 TiDB 集群支持两种备份策略：
+
+* [全量备份](#全量备份)（定时执行或 Ad-hoc）：使用 [`mydumper`](https://www.pingcap.com/docs/dev/reference/tools/mydumper/) 获取集群的逻辑备份；
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
* [全量备份](#全量备份)（定时执行或 Ad-hoc）：使用 [`mydumper`](/reference/tools/mydumper.md) 获取集群的逻辑备份；
```

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 6]
```diff
@@ -0,0 +1,140 @@
...
+* [增量备份](#增量备份)：使用 [`TiDB Binlog`](https://www.pingcap.com/docs/dev/reference/tools/tidb-binlog/overview/) 将 TiDB 集群的数据实时复制到其它数据库中或实时获得增量数据备份；
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
* [增量备份](#增量备份)：使用 [`TiDB Binlog`](/reference/tools/tidb-binlog/overview.md) 将 TiDB 集群的数据实时复制到其它数据库中或实时获得增量数据备份；
```

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 7]
```diff
@@ -0,0 +1,140 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm upgrade <release_name> pingcap/tidb-cluster -f values.yaml --version=<tidb-operator-version>
+    ```
+
+### Ad-hoc 全量备份
+
+Ad-hoc 全量备份封装在 `pingcap/tidb-backup` 这个 Helm chart 中。根据 `values.yaml` 文件中的 `mode`  配置，该 chart 可以执行全量备份或数据恢复。我们会在 [数据恢复](#数据恢复) 一节中描述如何执行数据恢复。
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
Ad-hoc 全量备份封装在 `pingcap/tidb-backup` 这个 Helm chart 中。根据 `values.yaml` 文件中的 `mode` 配置，该 chart 可以执行全量备份或数据恢复。我们会在 [数据恢复](#数据恢复) 一节中描述如何执行数据恢复。
```

File: v3.0/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 8]
```diff
@@ -0,0 +1,140 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm install pingcap/tidb-backup --namespace=<namespace> --name=<restore-name> -f values.yaml --version=<tidb-operator-version>
+    ```
+
+## 增量备份
+
+增量备份使用 [TiDB Binlog](https://www.pingcap.com/docs/dev/reference/tools/tidb-binlog/overview/) 工具从 TiDB 集群收集 Binlog，并提供实时备份和向其它数据库的实时同步能力。
```
Comment (yikeke) [inline v3.0/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
增量备份使用 [TiDB Binlog](/reference/tools/tidb-binlog/overview.md) 工具从 TiDB 集群收集 Binlog，并提供实时备份和向其它数据库的实时同步能力。
```

**Comment by aylei** [comment]
@yikeke Addressed PTAL

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 9]
```diff
@@ -0,0 +1,140 @@
+---
+title: Kubernetes 上的 TiDB 集群备份恢复
+category: how-to
+---
+
+# Kubernetes 上的 TiDB 集群备份与恢复
+
+## 概览
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
```

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 10]
```diff
@@ -0,0 +1,140 @@
+---
+title: Kubernetes 上的 TiDB 集群备份恢复
+category: how-to
+---
+
+# Kubernetes 上的 TiDB 集群备份与恢复
+
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
```

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 11]
```diff
@@ -0,0 +1,140 @@
...
+    helm upgrade <release_name> pingcap/tidb-cluster -f values.yaml --version=<tidb-operator-version>
+    ```
+
+### Ad-hoc 全量备份
+
+Ad-hoc 全量备份封装在 `pingcap/tidb-backup` 这个 Helm chart 中。根据 `values.yaml` 文件中的 `mode` 配置，该 chart 可以执行全量备份或数据恢复。我们会在 [数据恢复](#数据恢复) 一节中描述如何执行数据恢复。
+
+你可以通过下面的步骤执行一次 Ad-hoc 全量备份：
+
+* 修改 `values.yaml`：
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
1. 修改 `values.yaml`：
```
有序步骤用有序列表 1. 2. 3. 这样的表示方法，请依次修改下面的几个列表项。

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 12]
```diff
@@ -0,0 +1,140 @@
...
+    * 将 `clusterName` 设置为目标 TiDB 集群名字；
+    * 将 `mode` 设置为 `backup`；
+    * 将 `storage.className` 设置为用于存储数据的 PV 的 `storageClass`；
+    * 根据数据量调整 `storage.size`；
+
+    > **注意：**
+    >
+    > 你必须将 Ad-hoc 全量备份使用的 PVC 的 [reclaim policy](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy) 设置为 `Retain` 来确保你的数据安全。
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
    > 你必须将 Ad-hoc 全量备份使用的 PV 的 [reclaim policy](https://kubernetes.io/docs/tasks/administer-cluster/change-pv-reclaim-policy) 设置为 `Retain` 来确保你的数据安全。
```

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 13]
```diff
@@ -0,0 +1,140 @@
...
+
+* 创建一个包含数据库用户名和密码的 Kubernetes [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) 该用户必须拥有数据备份所需的数据库相关权限，同时，将 `values.yaml` 中的 `secretName` 设置为该 `Secret` 的名字（默认为 `backup-secret`）：
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
* 创建一个包含数据库用户名和密码的 Kubernetes [Secret](https://kubernetes.io/docs/concepts/configuration/secret/)，该用户必须拥有数据备份所需的数据库相关权限，同时，将 `values.yaml` 中的 `secretName` 设置为该 `Secret` 的名字（默认为 `backup-secret`）：
```

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 14]
```diff
@@ -0,0 +1,140 @@
...
+kubectl get pvc -n <namespace> -l app.kubernetes.io/component=backup,pingcap.com/backup-cluster-name=<cluster-name>
+```
+
+假如你将数据存储在 [Google Cloud Storage](https://cloud.google.com/storage/)，[Ceph Object Storage](https://ceph.com/ceph-storage/object-storage/) 或 [Amazon S3](https://aws.amazon.com/s3/) 中，你可以用这些存储系统自带的图形界面或命令行工具查看全量备份。
+
+## 数据恢复
+
+ 使用 `pingcap/tidb-backup` 这个 Helm chart 进行数据恢复，步骤如下：
+
+* 修改 `values.yaml`：
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
1. 修改 `values.yaml`：
```
依次修改下面的列表项。

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 15]
```diff
@@ -0,0 +1,140 @@
...
+    * 将 `clusterName` 设置为目标 TiDB 集群名；
+    * 将 `mode` 设置为 `restore`；
+    * 将 `name`  设置为用于恢复的备份名字（你可以参考[查看备份](#查看备份)来寻找可用的备份数据）。假如备份数据存储在 [Google Cloud Storage](https://cloud.google.com/storage/)，[Ceph Object Storage](https://ceph.com/ceph-storage/object-storage/) 或 [Amazon S3](https://aws.amazon.com/s3/) 中，你必须保证这些存储的相关配置与执行[全量备份](#全量备份)时一致。
+* 创建一个包含数据库用户名和密码的 Kubernetes [Secret](https://kubernetes.io/docs/concepts/configuration/secret/) 该用户必须拥有数据备份所需的数据库相关权限，同时，将 `values.yaml` 中的 `secretName` 设置为该 `Secret` 的名字（默认为 `backup-secret`，假如你在[全量备份](#全量备份)时已经创建了该 Secret，则可以跳过这步）：
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
* 创建一个包含数据库用户名和密码的 Kubernetes [Secret](https://kubernetes.io/docs/concepts/configuration/secret/)，该用户必须拥有数据备份所需的数据库相关权限，同时，将 `values.yaml` 中的 `secretName` 设置为该 `Secret` 的名字（默认为 `backup-secret`，假如你在[全量备份](#全量备份)时已经创建了该 Secret，则可以跳过这步）：
```

File: dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md [hunk 16]
```diff
@@ -0,0 +1,140 @@
...
+
+    {{< copyable "shell-regular" >}}
+
+    ```shell
+    helm install pingcap/tidb-backup --namespace=<namespace> --name=<restore-name> -f values.yaml --version=<tidb-operator-version>
+    ```
+
+## 增量备份
+
+增量备份使用 [TiDB Binlog](https://www.pingcap.com/docs/dev/reference/tools/tidb-binlog/overview/) 工具从 TiDB 集群收集 Binlog，并提供实时备份和向其它数据库的实时同步能力。
```
Comment (yikeke) [inline dev/how-to/maintain/tidb-in-kubernetes/backup-and-restore.md]:
```suggestion
增量备份使用 [TiDB Binlog](/reference/tools/tidb-binlog/overview.md) 工具从 TiDB 集群收集 Binlog，并提供实时备份和向其它数据库的实时同步能力。
```

**Pull request merged**